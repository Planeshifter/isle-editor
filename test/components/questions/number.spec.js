// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import Session from '@isle-project/session';
import Provider from '@isle-project/components/internal/provider';
import NumberQuestion from '@isle-project/components/questions/number/main.js';


// VARIABLES //

const session = new Session( {}, true );


// TESTS //

describe( '<NumberQuestion />', function test() {
	it( 'renders a number question', () => {
		const editor = renderer.create( <Provider session={session}>
			<NumberQuestion />
		</Provider> );
		expect( editor ).toMatchSnapshot();
	});
});
