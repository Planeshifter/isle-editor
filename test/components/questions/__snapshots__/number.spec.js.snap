// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NumberQuestion /> renders a number question 1`] = `null`;
